geode-core/src/test/java/org/apache/geode/cache/client/internal/ConnectionPoolImplJUnitTest.java

```diff
@@ -267,4 +270,12 @@ public boolean useThreadLocalConnection() {
     assertEquals(location1, pool.executeOnQueuesAndReturnPrimaryResult(testOp));
   }
 
+  @Test
+  public void testAddVarianceToInterval() {
+    assertThat(PoolImpl.addVarianceToInterval(0)).as("Zero gets zero variance").isEqualTo(0);
+    assertThat(PoolImpl.addVarianceToInterval(300000)).as("Large value gets +/-10% variance")
+        .isNotEqualTo(300000).isGreaterThanOrEqualTo(270000).isLessThanOrEqualTo(330000);
```

Comment:
This test will fail about 1 in 30000 times when it returns 300000.  Admittedly rare, but could be annoying when it happens.

Reply:
[I'm looking at older PRs] Would it be correct to just remove `.isNotEqualTo(300000)`?

Reply:
The change for the previous comment should prevent that.